### ISSUE TITLE
TCTS model

### ISSUE THREAD

**Issue by douhongkun** [microsoft/qlib#537]
Thanks for your contribution. But I have two questions about the TCTS model when i read its code:

Firstly, when updating the parameters of the weight model, the logarithmic probability should be based on the training data. That's what the algorithm diagram in this paper says. Why is the logarithmic probability on the verification set used when implementing here? 

Secondly, when using the verification set error as the reward, the opposite number should be used as the reward, And because pytorch drops in the optimization gradient, the loss function also produces a negative sign. The two negative signs should offset each other. On the contrary, why there is a negative sign in the source code. 

I don't know if there is any problem with my understanding. I hope I can get your answer.

![image](https://user-images.githubusercontent.com/77845036/126979235-2f57dca1-a022-4cb7-b856-4c4b25e76bbc.png)

**Comment by you-n-g** [comment]
@lwwang1995  Could you please help to answer this question? 
Thanks

**Comment by urnotLeo** [comment]
Also, may I ask why the **valid_loss** is computed by the **label[: , 0]**, does it mean now the main task is the next 1 day prediction?

**Comment by lwwang1995** [comment]
1. For the first question, please refer to the 10th line of the algorithm in the paper. The scheduler is optimized based on the validation performance.
2. The scheduler's loss had a bug during code refactoring, and it is now being repaired. We will update the experimental result soon.
3. The valid_loss should be computed by the label[: , target_label]. This bug is also being fixed.
